examples/mobilenet2_example.py

```diff
@@ -0,0 +1,98 @@
+import torch
+from torch import nn
+from torchvision import transforms, datasets
+from torch.utils.data import DataLoader
+
+from tqdm import tqdm
+
+from bitnet.nn.bitlinear import BitLinear
+from bitnet.nn.bitconv2d import BitConv2d
+from bitnet.models.resnet import resnet18
+from bitnet.models.mobilenet2 import create_mobilenet2, create_bit_mobilenet2
+from seed import set_seed
+
+
+device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+
+
+def train_model(
+        model: nn.Module,
+        train_loader: DataLoader,
+        optimizer: torch.optim.Optimizer,
+        criterion: nn.CrossEntropyLoss,
+        num_epochs: int) -> None:
+    for epoch in range(num_epochs):
+        pbar = tqdm(total=len(train_loader), desc=f"Training {model.name}")
+        running_loss: float = 0.0
+        for i, (images, labels) in enumerate(train_loader):
+            images, labels = images.to(device), labels.to(device)
+            outputs = model(images)
+            loss = criterion(outputs, labels)
+
+            optimizer.zero_grad()
+            loss.backward()
+            optimizer.step()
+
+            running_loss += loss.item()
+            pbar.set_description(
+                f'Model: {model.name} - Epoch [{epoch+1}], Loss: {running_loss / (i+1):.4f}'
+            )
+            pbar.update(1)
+        pbar.close()
+
+
+def test_model(model: nn.Module, test_loader: DataLoader):
+    model.eval()
+    correct: int = 0
+    total: int = 0
+    with torch.no_grad():
+        for images, labels in test_loader:
+            images, labels = images.to(device), labels.to(device)
+            outputs = model(images)
+            _, predicted = torch.max(outputs.data, 1)
+            total += labels.size(0)
+            correct += (predicted == labels).sum().item()
+    accuracy = 100 * correct / total
+    print(f'Accuracy of {model.name}: {accuracy:.2f}%')
+
+
+def main():
+    num_classes: int        = 100
+    learning_rate: float    = 1e-3
+    num_epochs: int         = 10
+    batch_size: int         = 128
+
+    transform = transforms.Compose([
+        transforms.Resize((224, 224)),
```

Comment:
CIFAR has an input size of 32x32, why are you resizing?
That's why it trains so slowly

Reply:
Because MNV2 was meant for that size - if I don't resize the accuracy drops down from 55% to 19% with BitNet and 61% to 27% for FloatNet. Do you think the drop in accuracy is acceptable?